app/views/sections/withdraw.html.erb

```diff
@@ -1,41 +1,89 @@
-<% content_for :page_title, "Withdraw section" %>
+<% content_for :title, "Withdraw section" %>
 
 <% content_for :breadcrumbs do %>
-  <li><%= link_to "Your manuals", manuals_path %></li>
-  <li><%= link_to manual.title, manual_path(manual) %></li>
-  <li><%= link_to section.title, manual_section_path(manual, section) %></li>
-  <li class='active'>Withdraw section</li>
+  <%= render "govuk_publishing_components/components/breadcrumbs", {
+    collapse_on_mobile: true,
+    breadcrumbs: [
+      {
+        title: "Your manuals",
+        url: manuals_path
+      },
+      {
+        title: manual.title,
+        url: manual_path(manual)
+      },
+      {
+        title: section.title,
+        url: manual_section_path(manual, section)
+      },
+      {
+        title: "Withdraw section"
+      }
+    ]
+  } %>
 <% end %>
 
-<h1 class="page-header">Withdraw section</h1>
-
-<div class='row'>
-  <div class="col-md-8">
-    <%= form_for [manual, section], method: :delete, class: 'panel panel-default' do |f| %>
-      <%= render partial: "shared/legacy_form_errors", locals: { object: section } %>
-
-      <div class="panel-body">
-        <p>This will remove this section from the manual.  This change will go
-          live when you publish the manual.  You will not be able to reinstate
-          it after you remove it.</p>
-
-        <div class="checkbox add-vertical-margins">
-          <%= f.radio_button :minor_update, 1, tag_type: :p, label: 'Minor update', checked: section.minor_update %>
-          <p class="help-block">Only use for minor changes like fixes to typos, links, GOV.UK style or metadata.</p>
-        </div>
-        <div class="checkbox add-vertical-margins">
-          <%= f.radio_button :minor_update, 0, tag_type: :p, label: 'Major update', checked: !section.minor_update %>
-        </div>
-        <div class="js-change-note-container <%= section.minor_update ? 'js-hidden' : nil %>">
-          <%= f.text_area :change_note, rows: 20, cols: 40, class: 'form-control short-textarea' %>
-          <p class="help-block">This will be publically viewable on GOV.UK.</p>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render "govuk_publishing_components/components/heading", {
+      text: "Withdraw section",
+      heading_level: 1,
+      font_size: "xl",
+      margin_bottom: 8
+    } %>
+  </div>
+</div>
+<div>
+  <p class="govuk-body">This will remove this section from the manual. This change will go live when you publish the manual.
```

Comment:
optional: On Whitehall we've used the warning component for messages like this 

https://components.publishing.service.gov.uk/component-guide/warning_text

Reply:
✅